packages/connectors/src/safe.ts

```diff
@@ -19,6 +19,13 @@ export type SafeParameters = Evaluate<
      * @default false
      */
     shimDisconnect?: boolean | undefined
+    /**
+     * Timeout in milliseconds for `getInfo` (from the Safe SDK) to resolve.
+     *
+     * `getInfo` does not resolve when not used in Safe App iFrame. This allows the connector to force a timeout.
+     * @default 10
```

Comment:
tested with `"Good 3G"` throttling on app.safe.global and has no problem resolving.

https://app.safe.global/apps/open?safe=eth%3A0x4557B18E779944BFE9d78A672452331C186a9f48&appUrl=http%3A%2F%2Flocalhost%3A5173